tests/tests_app/core/test_lightning_app_checkpoint.py

```diff
@@ -0,0 +1,146 @@
+import json
+import os
+from unittest import mock
+
+import pytest
+from tests_app import _PROJECT_ROOT
+
+from lightning_app import LightningApp, LightningFlow, LightningWork  # F401
+
+
+class SavedAppExample(LightningFlow):
```

Comment:
Could you test out with an app where the components take input arguments? This is usually the main issue.

Reply:
added 👍